### DISCUSSION TITLE
Welcome! 🤗

### DISCUSSION THREAD

**Discussion by brys0** [Shadfin/app#1] [Announcements]
### Discussions <i><ins>should</ins></i> be used for:
  * Features/Ideas ✨
  * Q/A ❓
  * Community 💬
  * Words of love 💖

### Discussions should <i><ins>not</ins></i> be used for:
* Reporting bugs 🐛
* Hate Speech/Put-downs 💢
* Talking about media acquisition 💿

In general if you can adhere to [Jellyfin's Community Standards](https://jellyfin.org/docs/general/community-standards/) you have nothing to worry about.

*Not affiliated or endorsed with Jellyfin or any of its subsidiaries*

**Comment by coder747-8i** [comment]
When can an IOS client be expected? Thank you

**Reply by brys0** [reply]
Thanks for your interest! Unfortunately I've just started college, and my time has been extremely limited but I've been working hard with learning compose with other projects.

I don't actually own a device with iOS, but I do own a Mac. Hopefully that'll be good enough to try and develop shadfin (with compose) for iOS.

I'm glad you're interested in my project, and thanks for the continued support.
